src/cli/backup/sharepoint.go

```diff
@@ -282,3 +314,94 @@ func deleteSharePointCmd(cmd *cobra.Command, args []string) error {
 
 	return nil
 }
+
+// ------------------------------------------------------------------------------------------------
+// backup details
+// ------------------------------------------------------------------------------------------------
+
+// `corso backup details onedrive [<flag>...]`
+func sharePointDetailsCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:     sharePointServiceCommand,
+		Short:   "Shows the details of a M365 SharePoint service backup",
+		RunE:    detailsSharePointCmd,
+		Args:    cobra.NoArgs,
+		Example: sharePointServiceCommandDetailsExamples,
+	}
+}
+
+// lists the history of backup operations
+func detailsSharePointCmd(cmd *cobra.Command, args []string) error {
+	ctx := cmd.Context()
+
+	if utils.HasNoFlagsAndShownHelp(cmd) {
+		return nil
+	}
+
+	s, acct, err := config.GetStorageAndAccount(ctx, true, nil)
+	if err != nil {
+		return Only(ctx, err)
+	}
+
+	r, err := repository.Connect(ctx, acct, s, options.Control())
+	if err != nil {
+		return Only(ctx, errors.Wrapf(err, "Failed to connect to the %s repository", s.Provider))
+	}
+
+	defer utils.CloseRepo(ctx, r)
+
+	opts := utils.SharePointOpts{
+		Sites:        site,
+		LibraryPaths: libraryPaths,
+		LibraryItems: libraryItems,
+
+		Populated: utils.GetPopulatedFlags(cmd),
+	}
+
+	ds, err := runDetailsSharePointCmd(ctx, r, backupID, opts)
+	if err != nil {
+		return Only(ctx, err)
+	}
+
+	if len(ds.Entries) == 0 {
+		Info(ctx, selectors.ErrorNoMatchingItems)
+		return nil
+	}
+
+	ds.PrintEntries(ctx)
+
+	return nil
+}
+
+// runDetailsSharePointCmd actually performs the lookup in backup details. Assumes
+// len(backupID) > 0.
```

Comment:
What is the consequence of this assumption?

Reply:
This comment is out of date.